packages/server/utils/AzureDevOpsServerManager.ts

```diff
@@ -0,0 +1,112 @@
+import fetch from 'node-fetch'
+import AzureDevOpsManager from 'parabol-client/utils/AzureDevOpsManager'
+import makeAppURL from 'parabol-client/utils/makeAppURL'
+import appOrigin from '../appOrigin'
+import {authorizeOAuth2} from '../integrations/helpers/authorizeOAuth2'
+import {
+  OAuth2PkceAuthorizationParams,
+  OAuth2PkceRefreshAuthorizationParams
+} from '../integrations/OAuth2Manager'
+import {IntegrationProviderAzureDevOps} from '../postgres/queries/getIntegrationProvidersByIds';
+// import getIntegrationProvidersByIds, {IntegrationProviderAzureDevOps} from '../postgres/queries/getIntegrationProvidersByIds';
+import {IGetTeamMemberIntegrationAuthQueryResult} from '../postgres/queries/generated/getTeamMemberIntegrationAuthQuery';
+import {isError} from 'util';
+
+class AzureDevOpsServerManager extends AzureDevOpsManager {
```

Comment:
-1 merge `AzureDevOpsManager` logic into this class

Reply:
Changed